src/main/java/org/jabref/logic/exporter/TemplateExporter.java

```diff
@@ -112,27 +102,33 @@ public TemplateExporter(String displayName, String consoleName, String lfFileNam
     }
 
     /**
-     * Initialize another export format based on templates stored in dir with
-     * layoutFile lfFilename.
-     * The display name is automatically derived from the FileType
+     * Initialize another export format based on templates stored in dir with layoutFile lfFilename.
      *
+     * @param displayName       Name to display to the user.
      * @param consoleName       Name to call this format in the console.
      * @param lfFileName        Name of the main layout file.
      * @param directory         Directory in which to find the layout file.
      * @param extension         Should contain the . (for instance .txt).
      * @param layoutPreferences Preferences for layout
      * @param savePreferences   Preferences for saving
-     * @param deleteBlankLines  If blank lines should be remove (default: false)
+     * @param deleteBlankLines  If blank lines should be removed (default: false)
      */
-    public TemplateExporter(String consoleName, String lfFileName, String directory, StandardFileType extension, LayoutFormatterPreferences layoutPreferences, SavePreferences savePreferences, boolean deleteBlankLines) {
-        this(consoleName, consoleName, lfFileName, directory, extension, layoutPreferences, savePreferences);
+    public TemplateExporter(String displayName, String consoleName, String lfFileName, String directory, FileType extension,
+                            LayoutFormatterPreferences layoutPreferences, SavePreferences savePreferences, boolean deleteBlankLines) {
```

Comment:
Can you make `boolean deleteBlankLines` an enum? `KEEP_BLANKS` or `DELETE_BLANKS` is way more intuitive as `true` and `false` at the caller side.